_health-care/_js/components/personal-information/VaMedicalFacility.jsx

```diff
@@ -11,6 +15,7 @@ import _ from 'lodash';
  * `value` - String. Stores the medical facility value.
  * `onValueChange` - a function with this prototype: (newValue)
```

Comment:
Now that we have a new prop for `VaMedicalFacility`, can we add a description here?